app/lib/pdf_filler/az140_pdf.rb

```diff
@@ -106,6 +106,7 @@ def hash_for_pdf
         "48" => @xml_document.at('Form140/DeductionAmt/SubTotal')&.text,
         "49" => @xml_document.at('DepTaxCredit')&.text,
         "50" => @xml_document.at('FamilyIncomeTaxCredit')&.text,
+        "51" => @xml_document.at('DeductionAmt CreditsFromAZ301')&.text,
```

Comment:
[DUST] Noticing here that there are variety of ways we're specifying the node that is nested within another one (see `Form140/DeductionAmt/SubTotal`) -- the directly sibling selectors (ex: `FamilyIncomeTaxCredit`, `DepTaxCredit`) are all not specified with `DeductionAmt` -- what do you think about adding the specificity to the other selectors, if we're going to do this for `CreditsFromAZ301`?

Reply:
ah, i don't mind adding the specificity to the other selectors. i prefer the specificity! 

tbh, i also did not realize that the specificity was not needed.

Reply:
just an update here! i refactored the subtraction nodes into their own section as can be seen on line 121 of this file. the only one not included is for "48." my theory is that Subtotal is used throughout the XML, and so there is a need to be specific by calling `Form140 DeductionAmt SubTotal`.

Reply:
also slashes seem to add more specificity (even opposed to spaces). i do not know why at this time.